Source/Dafny/Resolver.cs

```diff
@@ -4079,6 +4079,9 @@ public class XConstraint
           case "NumericOrBitvector":
             satisfied = t.IsNumericBased() || t.IsBitVectorType;
             break;
+          case "NumericOrBitvectorOrChar":
+            satisfied = t.IsNumericBased() || t.IsBitVectorType || t.IsCharType;
+            break;
```

Comment:
Is the new name `"NumericOrBitvectorOrChar"` ever used?

Is the old `"NumericOrBitvector"` name still used?

Reply:
Removed the case NumericOrBitvectorOrChar; NumericOrBitvector is still used.